core/templates/pages/exploration-editor-page/modal-templates/exploration-metadata-modal.controller.ts

```diff
@@ -29,6 +29,12 @@ require(
   'pages/exploration-editor-page/services/exploration-objective.service.ts');
 require('pages/exploration-editor-page/services/exploration-states.service.ts');
 require('pages/exploration-editor-page/services/exploration-tags.service.ts');
+require(
```

Comment:
So, weren't the corresponding directives imported here before? If not, how  did it work before?

Reply:
No, it wasn't. I didn't check before the import if it was really working, however, I've faced a situation where the application was breaking after changing the directive to component and the fix was to import it in its respective ts files. So I think this is just a double-check after migrating the files.